fix output of composer --version

before:
Composer version @package_branch_alias_version@ (1.0.0-alpha11) 2015-11-14 16:21:07

after:
Composer version 1.0.0-alpha11 2015-11-14 16:21:07

--- composer-1.0.0-17.beta1.1/src/Composer/Composer.php~	2016-03-03 17:15:10.000000000 +0200
+++ composer-1.0.0-17.beta1.1/src/Composer/Composer.php	2016-03-21 14:51:35.024715454 +0200
@@ -29,7 +29,7 @@
 class Composer
 {
     const VERSION = '1.0.0-beta1';
-    const BRANCH_ALIAS_VERSION = '@package_branch_alias_version@';
+    const BRANCH_ALIAS_VERSION = '';
     const RELEASE_DATE = '2016-03-03 15:15:10';
 
     /**
